packages/blocksAntd/src/blocks/Statistic/Statistic.json

```diff
@@ -0,0 +1,73 @@
+{
+  "category": "display",
+  "loading": false,
+  "schema": {
+    "properties": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "decimalSeparator": {
+          "type": "string",
+          "default": ".",
+          "description": "Decimal separator."
+        },
+        "formatter": {
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string",
+              "enum": ["default", "date", "humanizeDuration"],
```

Comment:
Options are:
- intlDateTimeFormat,
- intlListFormat,
- intlNumberFormat,
(@lowdefy/format). Properties are different for each formatter
- intlRelativeTimeFormat,
- momentFormat,
- momentHumanizeDuration,